src/oaklib/converters/owl_to_fhir.py

```diff
@@ -0,0 +1,20 @@
+"""Convert OWL to FHIR CodeSystem JSON
+
+TODO's
```

Comment:
Some TODO's
- [ ] 1. Write functioning converter
- [ ] 2. Optional/Later: Schema mappings (Obograph -> FHIR CodeSystem). SchemaAutomator? Other approach?
- [ ] 3. Add converter to CLI
- [ ] 4. Test(s)